### PR TITLE
Use original case on buttons/tabs

### PR DESCRIPTION
**Pull request by tstirrat**
Screen Shot 2020-03-11 at 12 09 01 PM

### CODE REVIEW

File: src/_material-overrides.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// for more specificity over originals
+.mdc-button.mdc-button {
```
Comment (yuchenshi) [inline src/_material-overrides.scss]: OMG this is ugly and I think we should do this instead:

```suggestion
.App .mdc-button {
```

And move to `App.scss` or `index.scss` if you don't mind.

Comment (tstirrat) [reply src/_material-overrides.scss]: I think we should keep material overrides in a single place. So there is clear obvious intent where we override the default material things. We may also need to do this for *all* select boxes etc. so on and so forth

Comment (tstirrat) [reply src/_material-overrides.scss]: Also "OMG this is ugly" could have been omitted ;)

Comment (yuchenshi) [reply src/_material-overrides.scss]: point taken

Comment (tjlav5) [reply src/_material-overrides.scss]: Is there any chance we can upgrade our material lib to use the latest mixins which includes text-transform: https://github.com/material-components/material-components-web/blame/master/packages/mdc-tab/_mixins.scss#L212

Comment (yuchenshi) [reply src/_material-overrides.scss]: It's a transitive dependency from rmwc so I'd say let's try upgrading to latest rmwc and then `npm update`. If it still doesn't get us there (probably won't for 4.x), then I'd not spend too much effort there.

Comment (tstirrat) [reply src/_material-overrides.scss]: Sadly, its v4+ and RMWC is still on 3.

Comment (tstirrat) [reply src/_material-overrides.scss]: https://github.com/jamesmfriedman/rmwc/blob/master/src/tabs/package.json#L35

**Pull request merged**